lib/config/config.js

```diff
@@ -70,41 +59,9 @@ class Config {
   }
 
   watch () {
-    const watchEvents = ['add', 'change'];
-
-    const watchOptions = {
```

Comment:
just change it to
    const watchOptions = {
      awaitWriteFinish: true,
      ignoreInitial: true
    };